scout/server/blueprints/variants/views.py

```diff
@@ -153,12 +159,15 @@ def variants(institute_id, case_name):
     if request.form.get("export"):
         return controllers.download_variants(store, case_obj, variants_query)
 
+    dismiss_options = return_dismissed_variants(case_obj.get("track"))
```

Comment:
You don't need to invoke a function here, you could just provide the DISMISS_VARIANT_OPTIONS when you return the dictionary (line 170)